tests/test_protocol_nmea.cpp

```diff
@@ -18,6 +20,65 @@ using namespace std;
 #define DEBUG_PRINTF	
 #endif
 
+
+void compareGpsPos(gps_pos_t &g1, gps_pos_t &g2)
+{
+    EXPECT_NEAR(g1.timeOfWeekMs, g2.timeOfWeekMs, 1);
+    EXPECT_EQ(g1.week, g2.week);
+    EXPECT_EQ(g1.status, g2.status);
+    for (int i=0; i<3; i++)
+    {
+        EXPECT_NEAR(g1.ecef[i], g2.ecef[i], 0.02);      // 20 cm
+    }
+    for (int i=0; i<2; i++)
+    {
+        EXPECT_NEAR(g1.lla[i], g2.lla[i], 1.0e-6);      // (deg)
+    }
+    EXPECT_NEAR(g1.lla[2], g2.lla[2], 0.01);            // 10 cm
+    EXPECT_NEAR(g1.hMSL, g2.hMSL, 0.01);                // 10 cm
+    EXPECT_NEAR(g1.hAcc, g2.hAcc, 0.01);                // 10 cm
+    EXPECT_NEAR(g1.vAcc, g2.vAcc, 0.01);                // 10 cm
+    EXPECT_NEAR(g1.pDop, g2.pDop, 0.01);
+    EXPECT_NEAR(g1.cnoMean, g2.cnoMean, 0.01);
+    EXPECT_NEAR(g1.towOffset, g2.towOffset, 0.01);
+    EXPECT_EQ(g1.leapS, g2.leapS);
+    EXPECT_EQ(g1.satsUsed, g2.satsUsed);
+    EXPECT_EQ(g1.cnoMeanSigma, g2.cnoMeanSigma);
+    EXPECT_EQ(g1.reserved, g2.reserved);
+}
+
+void compareGpsVel(gps_vel_t &g1, gps_vel_t &g2)
+{
+    EXPECT_NEAR(g1.timeOfWeekMs, g2.timeOfWeekMs, 1);
+    for (int i=0; i<3; i++)
+    {
+        EXPECT_NEAR(g1.vel[i], g2.vel[i], 0.01);        // 10 cm
+    }
+    EXPECT_NEAR(g1.sAcc, g2.sAcc, 0.01);                // 10 cm/s
+    EXPECT_EQ(g1.status, g2.status);
+}
+
+// TEST(protocol_nmea, gps_to_utc_time_conversion)
```

Comment:
Dead code?  Or test that wouldn't pass but eventually needs to be fixed?   If the later, a FIXME should be added with a comment; otherwise, let's kill the dead code.